### PR TITLE
Add enable systemd-coredump by default post

### PR DESCRIPTION
**Pull request by mhabrnal**
Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>

### CODE REVIEW

File: _posts/2017-03-07-fedora-26-change.markdown [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+It means, core dumps will be processed by `systemd-coredump` and metadata about
+core dumps will be stored in the systemd journal. As I said `abrt-ccpp.service`
+won't be enabled but `abrt-journal-core.service` will be enabled instead.
+The service extracts coredumps from systemd journal and other important
+information about problems.
+
+# What changes it brings for ABRT's users?
+ABRT users shouldn't experience any changes. Problem directories will be still
+created in `/var/spool/abrt` and all will work like in older Fedoras.
+If users want use ABRT's hook instead of the systemd ones they can easily
```
Comment (jfilak) [inline _posts/2017-03-07-fedora-26-change.markdown]: s/want/want to/

File: _posts/2017-03-07-fedora-26-change.markdown [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# How it worked?
+In older Fedoras there is a service `abrt-ccpp.service` which runs
+`/usr/sbin/abrt-install-ccpp-hook install` which causes ABRT's
+`kernel.core_pattern` hook is installed.
+ Basicaly, current `kernel.core_pattern` is stored in
+`/var/run/abrt/saved_core_pattern` and `kernel.core_pattern =
+|/usr/libexec/abrt-hook-ccpp %s %c %p %u %g %t %P %I` it's used instead.
+This causes all coredumps are processed by ABRT's hook which by default dumps
+core into ABRT's problem directories which are created in `/var/spool/abrt`.
+After you can use `abrt` command line tool for further analysis (remote
```
Comment (xsuchy) [inline _posts/2017-03-07-fedora-26-change.markdown]: After that ...

**Comment by mhabrnal** [comment]
The comments are addressed.

**Comment by marusak** [comment]
I rewrote it a bit to be better readable also for not very skilled people with ABRT and Linux like myself:)

File: _posts/2017-03-07-fedora-26-change.markdown [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+categories: abrt systemd core_pattern
+---
+Enabling systemd-coredump by default is from the perspective of ABRT the
+biggest change in Fedora 26.
+
+# How ABRT works?
+There is a service `abrt-ccpp.service` which runs
+`/usr/sbin/abrt-install-ccpp-hook install` that causes ABRT's
+`kernel.core_pattern` hook to be installed. What that basically does
+is it saves current `kernel.core_pattern` to `/var/run/abrt/saved_core_pattern`
```
Comment (xsuchy) [inline _posts/2017-03-07-fedora-26-change.markdown]: What it basically does is that it saves... IIMHO

File: _posts/2017-03-07-fedora-26-change.markdown [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title:  "ABRT in Fedora 26 with systemd-coredump"
+date:   2017-03-07 12:48:44 +0100
+author: Matej Habrnal <mhabrnal@redhat.com>
+visible: 1
+categories: abrt systemd core_pattern
+---
+Enabling systemd-coredump by default is from the perspective of ABRT the
```
Comment (xsuchy) [inline _posts/2017-03-07-fedora-26-change.markdown]: Can you link https://fedoraproject.org/wiki/Changes/coredumpctl here?

**Comment by marusak** [comment]
@xsuchy done

**Comment by jfilak** [comment]
You forget to mention that with this change ABRT will no longer generate backtrace from the process being killed and thus quality of u-reports on FAF might fall down a bit. I would appeal to all who cares about reporting crashes to switch back to abrt-ccpp.service.

**Pull request merged**